client/packages/components/packages-list/style.scss

```diff
@@ -0,0 +1,107 @@
+.packages-list {
+	div {
+		box-sizing: border-box;
+	}
+
+	&.form-fieldset {
+		margin-bottom: 12px;
+		padding-bottom: 10px;
+		border-bottom: 1px solid lighten( $gray, 30% );
+	}
+}
+
+.packages-list__column-type {
+	width: 10%;
+	text-align: center;
+	@include breakpoint( '<660px' ) {
+		width: 11%;
+	}
+
+	.packages-list__type-icon {
+		fill: $gray;
+	}
+}
+
+.packages-list__column-name {
+	width: 46%;
+	padding-right: 8px;
+	@include breakpoint( '<660px' ) {
+		width: 44%;
+	}
+
+	&.form-legend {
+		padding-right: 0;
+	}
+
+	.packages-list__no-name {
+		font-style: italic;
+	}
+}
+
+.packages-list__column-dimensions {
+	width: 39%;
+	cursor: default;
+	@include breakpoint( '<660px' ) {
+		width: 37%;
+	}
+}
+
+.packages-list__column-actions {
+	width: 5%;
+	text-align: right;
+	@include breakpoint( '<660px' ) {
+		width: 8%;
+	}
+}
+
+.packages-list__empty {
+	padding: 8px 0;
+}
+
+.packages-list__empty-icon .gridicon {
+	margin-right: 8px;
+	margin-bottom: -4px;
+	fill: $gray;
+}
+
+.packages-list__empty-description {
+	color: $gray;
+}
+
+.packages-list__header {
+	display: flex;
+	height: 26px;
+	border-bottom: 1px solid lighten( $gray, 30% );
+}
+
+.packages-list__item,
+.packages-list__empty {
+	margin-top: 8px;
+	display: flex;
+	//align-items: center;
```

Comment:
Removed in https://github.com/Automattic/woocommerce-services/pull/1051/commits/6028de9a090e8076d9c18dc57db9162f48ec4b7a